### PR TITLE
Lint fixes

### PR DESCRIPTION
**Pull request by jamesbeedy**
I haven't deployed this, might want to give it a few trial runs before merging :-)

### CODE REVIEW

**Comment by galgalesh** [comment]
Awesome, thanks!

You changed the built charm itself, not the corresponding layers, but I'm working on the charm right now, so I'll merge the pull request and copy the changes myself.

Which linter do you use? I'm using pylint at the moment but I disable some of the checks using #pylint: disable=...

File: charms/xenial/sojobo-api/config.yaml [hunk 1]
```diff
@@ -28,7 +29,8 @@
       \ key is used that will already be installed on the machine, and for PPA sources\
       \ where the package signing key is securely retrieved from Launchpad.\n"
     "type": "string"
-    "default": ""
+    "default": |
+       - "C8068B11"
```
Comment (galgalesh) [inline charms/xenial/sojobo-api/config.yaml]: I suspect this is the key of the ppa? How do you find such a key?

Comment (jamesbeedy) [inline charms/xenial/sojobo-api/config.yaml]: Select the "Technical details about this PPA" dropdown here -> https://launchpad.net/~juju/+archive/ubuntu/stable

Also line 8 here -> http://paste.ubuntu.com/23454428/

**Comment by jamesbeedy** [comment]
@galgalesh yeah, I was wondering why they were built charms .... why do you version the built charms in github,  any reason specifically?

**Comment by galgalesh** [comment]
@jamesbeedy 

1. We want to have a known-to-be-good built charm laying around. The `charm build` process isn't reproducible since upstream layers and the reactive framework change and there isn't a way to use previous versions of those.

2. Bugs in earlier versions of `charm build` and the reactive framework required manual patches to the built charm, although all these bugs have been fixed now.

**Pull request merged**